pkgs/top-level/all-packages.nix

```diff
@@ -5209,6 +5209,10 @@ in
 
   minetime = callPackage ../applications/office/minetime { };
 
+  mini-graph-card = callPackage ../servers/home-assistant/lovelaceModules/mini-graph-card {};
+
+  mini-media-player = callPackage ../servers/home-assistant/lovelaceModules/mini-media-player {};
```

Comment:
I would actually group packages like that into a common attribute set for better discoveribility, maybe hassCustomCards?
It could be similar to `vimPlugins` or `mpvScripts` or `passExtension`.